package.json

```diff
@@ -37,20 +37,20 @@
   },
   "homepage": "https://github.com/chanzuckerberg/edu-design-system#readme",
   "devDependencies": {
-    "@commitlint/cli": "^8.3.5",
-    "@commitlint/config-conventional": "^8.3.4",
-    "@commitlint/prompt-cli": "^8.3.5",
+    "@commitlint/cli": "^11.0.0",
+    "@commitlint/config-conventional": "^11.0.0",
+    "@commitlint/prompt-cli": "^11.0.0",
     "@size-limit/preset-small-lib": "^4.9.2",
     "@typescript-eslint/eslint-plugin": "^4.15.1",
     "@typescript-eslint/parser": "^4.15.1",
     "codecov": "^3.8.1",
-    "eslint": "^6.8.0",
-    "eslint-config-prettier": "^6.15.0",
+    "eslint": "^7.20.0",
+    "eslint-config-prettier": "^8.0.0",
     "eslint-plugin-jest": "^22.21.0",
```

Comment:
Didn't update `eslint-plugin-jest`, because the latest version (v24) can't figure out what our version of Jest is in our setup.